scanomatic/image_analysis/image_grayscale.py

```diff
@@ -161,8 +162,8 @@ def get_para_trimmed_slice(
 
 
 def get_grayscale(
-    fixture,
-    grayscale_area_model,
+    fixture,  # FixtureImage (but circular import)
```

Comment:
Could `get_grayscale` move to a different/own file? Worth an issue?

Reply:
It should, or something else should move... I'll make an issue

Reply:
there's also another function in another file that is called `get_grayscale` and it's a bit confusing.

Reply:
https://github.com/Scan-o-Matic/scanomatic-standalone/issues/118